src/convertToObject.js

```diff
@@ -3,16 +3,26 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (
+ * see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesCSS = {};
+  const arrayOfStyles = sourceString
+    .split(';')
+    .filter(item => item.trim())
+    .map(item => item.split(':'));
+
+  arrayOfStyles.forEach(item => (stylesCSS[item[0].trim()] = item[1].trim()));
```

Comment:
Create meaningful variables, `item` here - is not a good option. Variable name should describe what inside it.  `stylesString` for example would be better.